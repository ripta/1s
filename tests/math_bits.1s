"math" {LOAD}

55 int->bits
[ 1 1 0 1 1 1 ] bits->int

55 int->bitstring

0 bit->bitstring
1 bit->bitstring
2 bit->bitstring

[ 1 1 0 0 0 1 0 ] bits->int
