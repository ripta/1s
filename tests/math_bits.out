Result:
  [ 1 1 0 1 1 1 ]
  55
  "110111"
  "0"
  "1"
  "1"
  98
